projects/plugins/crm/src/automation/class-automation-bootstrap.php

```diff
@@ -34,51 +34,13 @@ final class Automation_Bootstrap {
 	public function init(): void {
 		$this->engine = Automation_Engine::instance();
 
-		$this->register_data_types();
```

Comment:
We don't need to register data_type along a slug. They are well-known elements (just use the class name).